src/java/org/apache/cassandra/index/sai/disk/format/Version.java

```diff
@@ -74,7 +74,7 @@ public class Version implements Comparable<Version>
     public static final Version VECTOR_EARLIEST = BA;
     public static final Version JVECTOR_EARLIEST = CA;
     public static final Version BM25_EARLIEST = EC;
-    public static final Version LATEST = EC;
+    public static final Version LATEST = ED;
```

Comment:
Not sure if a comment saying `max existing version, not to confuse with CURRENT` or something like that would help. Or perhaps maybe making it relative to `ALL`:
```
public static final Version LATEST = ALL.get(0);
```
WDYT? Just wondering, not asking to change it here.

Reply:
Makes sense, I will change it here so we do not repeat again, missing to update it. Thanks